test/components/plots/barPlotTests.ts

```diff
@@ -64,18 +64,18 @@ describe("Plots", () => {
         var bar1 = d3.select(bars[0][1]);
         assert.closeTo(TestMethods.numAttr(bar0, "width"), xScale.rangeBand(), 1, "bar0 width is correct");
         assert.closeTo(TestMethods.numAttr(bar1, "width"), xScale.rangeBand(), 1, "bar1 width is correct");
-        assert.equal(bar0.attr("height"), "100", "bar0 height is correct");
-        assert.equal(bar1.attr("height"), "150", "bar1 height is correct");
+        assert.strictEqual(+bar0.attr("height"), 100, "bar0 height is correct");
+        assert.strictEqual(+bar1.attr("height"), 150, "bar1 height is correct");
         assert.closeTo(TestMethods.numAttr(bar0, "x"), 111, 1, "bar0 x is correct");
```

Comment:
As @jtlan was nicely pointing out to me at some point, we should think of our testing as a set of axioms which are build on each other. This means that we assume that a height attribute of `"100"` is to correspond to its actual height. (if this is not yet covered, it should be).

I can definitely see your point here, but I can't go over the fact that `+null === 0`, which will make some of the tests (see below) useless. See for yourself
```typescript
+d3.select('svg').attr('foo') === 0 // returns true
```

I believe that we are being too pedantic about semantics in this case. The tests have to be correct in the first place. Then we can be pedantic about them.